src/rust/pal/functions.rs

```diff
@@ -16,6 +42,18 @@ pub fn get_addr_from_sock_addr_in(sock_addr_in: &SockAddrIn) -> u32 {
 // Linux functions
 //======================================================================================================================
 
+#[cfg(all(feature = "catnip-libos", target_os = "linux"))]
+pub fn create_sin_addr(octets: &[u8; 4]) -> in_addr {
+    in_addr {
+        s_addr: u32::from_le_bytes(*octets),
+    }
+}
+
+#[cfg(all(feature = "catnip-libos", target_os = "linux"))]
+pub fn create_sin_zero() -> [u8; 8] {
+    [0; 8]
+}
```

Comment:
This is not specific to Linux, thus it should go in the PAL module, but not inside the "platform-dependent" submodule.

Reply:
Hi @ppenna, this is indeed Linux specific code. For Windows, this code is just CHAR(0). Or maybe I'm not getting your comment.

Reply:
@anandbonde, I do understand that this code is only used in Windows. But fundamentally it is not specific to this platform, thus you could place it in a PAL layer that is common to both. Rationale:
- In the PAL windows module we find only platform-specific code.
- If Linux starts requiring this to be defined in the future we don't end up with duplicates
- If we introduce support for other LibOS that does require the same definition, we already have it.

Reply:
@anandbonde I got your point. I was reading `CHAR(0)` as u8.